hermes/crates/cbork/cddl-parser/tests/comments.rs

```diff
@@ -3,53 +3,58 @@ use cddl_parser::{
     cddl_test::{CDDLTestParser, Parser, Rule},
 };
 
+pub(crate) const COMMENT_PASSES: &[&str] = &["; A Comment \n", "; And another\r", ";more\r\n"];
+
+pub(crate) const COMMENT_FAILS: &[&str] = &["not a comment\n"];
+
+pub(crate) const WHITESPACE_COMMENT_PASSES: &[&str] = &[
+    " ",
+    "  ",
+    " \t \t",
+    " \t  \r \n \r\n   ",
+    "; A Comment\r",
+    " \t ; A Comment    \n",
+    "; One Comment\n; Two Comments\n",
+    "; One Comment  \n; Two Comments\r; Another Comment\r\n",
+    "\t; One Comment \n\t; Two Comments\r; Another Comment\r\n",
+    "\t; A Comment \n    ; Another Comment \t \r\n    \t  ; A Final Comment   \r\n",
+];
+
+pub(crate) const WHITESPACE_COMMENT_FAILS: &[&str] = &["not a comment"];
+
+/// # Panics
+pub(crate) fn passes_tests_rule(rule_type: Rule, test_data: &[&str]) {
+    for test in test_data {
+        let parse = CDDLTestParser::parse(rule_type, test);
+        assert!(parse.is_ok());
+    }
+}
```

Comment:
DRY.
Define these utilities once, in one file, and import them when you need to use them.